superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -726,22 +737,27 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
   }, [getChartData, noChartLabel]);
 
   const loadChartOptions = useMemo(
-    () => (input = '', page: number, pageSize: number) => {
-      const query = rison.encode({ filter: input, page, page_size: pageSize });
-      return SupersetClient.get({
-        endpoint: `/api/v1/report/related/chart?q=${query}`,
-      }).then(response => {
-        const list = response.json.result.map(
-          (item: { value: number; text: string }) => ({
-            value: item.value,
-            label: item.text,
-          }),
-        );
-
-        setChartOptions(list);
-        return { data: list, totalCount: response.json.count };
-      });
-    },
+    () =>
```

Comment:
Sorry I'm noting a lot of things not related to your PR, but this block is repeated a few times, and we could DRY it up later.